_pages/en_US/ntrboot.txt

```diff
@@ -23,8 +23,8 @@ Note that carts with a "Time Bomb" will no longer be able to launch `.nds` files
 | [**DSTT**](http://www.nds-card.com/ProShow.asp?ProID=157) | $9.99 | No | None | None | Only models with [certain flash chips](https://gist.github.com/aspargas2/fa2a70aed3a7fe33f1f10bc264d9fab6) are compatible with ntrboot. |
 | [**R4i Gold 3DS**](http://www.nds-card.com/ProShow.asp?ProID=149) | $19.99 | No | <= 11.14.0 | <= 1.4.5 | All RTS revisions are compatible. |
 | [**R4i-SDHC 3DS RTS**](http://www.nds-card.com/ProShow.asp?ProID=146) | $13.99 | 1.85b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | |
-| [**R4iSDHC GOLD Pro 20XX**](http://www.nds-card.com/ProShow.asp?ProID=490) | $9.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013 models are incompatible. |
-| [**R4iSDHC RTS LITE 20XX**](http://www.nds-card.com/ProShow.asp?ProID=450) | $13.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013 models are incompatible. |
+| [**R4iSDHC GOLD Pro 20XX**](http://www.nds-card.com/ProShow.asp?ProID=490) | $9.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013, 2021 and later models are incompatible. |
+| [**R4iSDHC RTS LITE 20XX**](http://www.nds-card.com/ProShow.asp?ProID=450) | $13.99 | 4.0b: September 3, 2024 | <= 11.14.0 | <= 1.4.5 | All R4iSDHC 20XX flashcarts are identical. 2013, 2021 and later models are incompatible. |
```

Comment:
Use "Only cartridges marked with a year between 2014 and 2020 are compatible." Not sure how to make that concise and to clarify that it's inclusive of both 2014 and 2020, though...